components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -121,6 +125,24 @@ export class TeamDBImpl implements TeamDB {
         });
     }
 
+    public async removeMemberFromTeam(userId: string, teamId: string): Promise<void> {
+        if (teamId.length !== 36) {
```

Comment:
nit: why does this matter?

Reply:
I like to program very defensively, but you're right, it doesn't really matter here. Removing.